### ISSUE TITLE
Encoding issue in VPack serializer

### ISSUE THREAD

**Issue by miguel-porto** [arangodb/arangodb-java-driver#70]
Hi, 
there seems to be an encoding issue in util().serialize(...). Take these lines, assuming that `newTaxEnt` has the string "açb" in field `author`
```
System.out.println(newTaxEnt.getAuthor());    // outputs açb
VPackSlice value = dbDriver.util().serialize(newTaxEnt, false);
System.out.println(value.get("author").getAsString());    // outputs aç
```
The actual value stored in the database is also truncated.
Is it a problem in the serializer, or am I assumed to encode the string in other way before passing it?
Best,
Miguel

**Comment by miguel-porto** [comment]
Maybe this has been fixed with #65 ?

**Comment by mpv1989** [comment]
Yes. I fixed it with version 4.1.1 which was released today.
